agents/types/dispute_processed_status.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+// DisputeProcessedStatus is the status of a dispute on Summit.
+// This enum is used for tracking the status of disputes in the Dispute db model.
+type DisputeProcessedStatus uint8
```

Comment:
shouldn't this be a db.Enum?

Reply:
What is the purpose of doing this? Seems like overhead. Does it help with querying in some way?